package.json

```diff
@@ -25,14 +25,14 @@
     "nunjucks": "^3.0.0"
   },
   "devDependencies": {
-    "babel-eslint": "^4.1.3",
+    "babel-eslint": "^7.2.3",
     "expect.js": "^0.3.1",
     "grunt": "^0.4.5",
-    "grunt-jscs": "^2.8.0",
-    "standard": "^5.3.1"
+    "snazzy": "^7.0.0",
+    "standard": "^10.0.2"
   },
   "scripts": {
-    "test": "node_modules/standard/bin/cmd.js && grunt test && node tests/all.js"
+    "test": "standard | snazzy && grunt test && node tests/all.js"
```

Comment:
Yeah, looks rad. Are you sure it's okay to use global names? I think it should be `./node_modules/.bin/(standart|snazzy)`, right?